service/chartconfig/v1/appr/appr.go

```diff
@@ -10,31 +18,97 @@ import (
 type Config struct {
 	Logger micrologger.Logger
 
-	Address string
+	Address      string
+	Organization string
 }
 
 // Client knows how to talk with a CNR server.
 type Client struct {
-	logger micrologger.Logger
+	httpClient *http.Client
+	logger     micrologger.Logger
 
-	address string
+	base *url.URL
 }
 
 // New creates a new configured appr client.
 func New(config Config) (*Client, error) {
 	if config.Logger == nil {
 		return nil, microerror.Maskf(invalidConfigError, "logger must not be empty")
 	}
-
 	if config.Address == "" {
 		return nil, microerror.Maskf(invalidConfigError, "address must not be empty")
 	}
+	if config.Organization == "" {
+		return nil, microerror.Maskf(invalidConfigError, "organization must not be empty")
+	}
+
+	// set client timeout to prevent leakages.
+	hc := &http.Client{
+		Timeout: time.Second * 5,
+	}
+
+	u, err := url.Parse(config.Address + "/" + config.Organization + "/")
```

Comment:
I agree that this is easy to do instead of using helm + the plugin. My only concern would be that an update of the plugin where this URL is different, would now break.
Nothing major, so I am fine with this.

Reply:
:+1: we need to keep an eye on the CNR specification and adapt to it as needed. Using the plugin we are exposed to the same problem though. I hope that any eventual change will be backward compatible.